certified-connectors/SmartDialog/apiDefinition.swagger.json

```diff
@@ -1,808 +1,1376 @@
+                                "properties": {
+                                    "inputSource": {
+                                        "type": "object",
+                                        "properties": {
+                                            "address": {
+                                                "type": "string",
+                                                "description": "WhatsApp identity number.",
+                                                "title": "Identity Number"
+                                            }
+                                        },
+                                        "description": "inputSource"
+                                    }
+                                }
+                            }
+                        }
+                    }
                 },
-                "ServiceType": "OneWay"
-              }
+                "operationId": "GetCustomerWhatsappIdentities",
+                "x-ms-visibility": "internal",
+                "summary": "Get Customer WhatsApp Identities",
+                "description": "Gets the Customers available WhatsApp identities.",
+                "parameters": [
+                    {
+                        "name": "Customer-Id",
+                        "in": "path",
+                        "required": true,
+                        "type": "string",
+                        "x-ms-url-encoding": "single",
+                        "x-ms-dynamic-values": {
+                            "operationId": "GetUserCustomers",
+                            "value-path": "customerId",
+                            "value-collection": "customerUsers",
+                            "value-title": "customerName"
+                        },
+                        "x-ms-visibility": "important"
+                    }
+                ]
             }
-          },
-          {
-            "name": "Request Body",
-            "in": "body",
-            "required": true,
-            "schema": {
-              "type": "object",
-              "properties": {
-                "TemplateName": {
-                  "type": "string",
-                  "description": "The Template to use for sending.",
-                  "title": "WhatsApp Template",
-                  "x-ms-visibility": "important",
-                  "x-ms-dynamic-values": {
-                    "operationId": "GetCustomerWhatsAppTemplates",
-                    "value-path": "id",
-                    "value-title": "extensionParameters/rawContent",
-                    "parameters": {
-                      "Customer-Id": {
-                        "parameter": "Customer-Id"
-                      }
+        },
+        "/whatsapp/opt-in/sender/{Sender}/recipient/{Recipient}": {
+            "delete": {
+                "responses": {
+                    "204": {
+                        "description": "default",
+                        "schema": {}
```

Comment:
@robin-larsson Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger.

Applies the same comment for all the empty schemas in the swagger